src/main/java/com/spotinst/sdkjava/model/SpotOceanEcsClusterService.java

```diff
@@ -0,0 +1,208 @@
+package com.spotinst.sdkjava.model;
```

Comment:
Create a Service folder under model and move the file under it in correct hierarchy - service.ocean.ecs